content/content.go

```diff
@@ -1,14 +1,86 @@
 package content
 
-const ContentPath string = "content"
+import (
+	"os"
+	"path/filepath"
+	"reflect"
+	"strings"
 
-type DependencyInformation struct {
-	Name      string `yaml:"name"`
-	SourceURI string `yaml:"source_uri"`
-	Hash      string `yaml:"hash"`
+	"github.com/pkg/errors"
+	uuid "github.com/satori/go.uuid"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	DefaultPath              = "gen/content"
+	DefaultMetadataFileName  = "metadata"
+	ActionSuffix             = ".action"
+	defaultHeader            = "# Code generated by openapi CLI. DO NOT EDIT.\n"
```

Comment:
There's no longer a CLI, let's change it to something that makes sense
```suggestion
	defaultHeader            = "# This file is generated. DO NOT EDIT.\n"
```